tests/endtoend/test_http_functions.py

```diff
@@ -112,14 +113,106 @@ class TestHttpFunctionsStein(TestHttpFunctions):
 
     @classmethod
     def get_script_dir(cls):
-        return testutils.E2E_TESTS_FOLDER / 'http_functions' /\
+        return testutils.E2E_TESTS_FOLDER / 'http_functions' / \
                                             'http_functions_stein'
 
 
 class TestHttpFunctionsSteinGeneric(TestHttpFunctions):
 
     @classmethod
     def get_script_dir(cls):
-        return testutils.E2E_TESTS_FOLDER / 'http_functions' /\
-                                            'http_functions_stein' /\
+        return testutils.E2E_TESTS_FOLDER / 'http_functions' / \
+                                            'http_functions_stein' / \
                                             'generic'
+
+
+class TestCommonLibsHttpFunctions(testutils.WebHostTestCase):
+    """Test the native Http Trigger in the local webhost.
+
+    This test class will spawn a webhost from your <project_root>/build/webhost
+    folder and replace the built-in Python with azure_functions_worker from
+    your code base. Since the Http Trigger is a native suport from host, we
+    don't need to setup any external resources.
+
+    this file is more focus on testing the E2E flow scenarios.
+    """
+
+    def setUp(self):
```

Comment:
You don't need the setup and teardown method

Reply:
updated. Thanks